### ISSUE TITLE
Option to ignore/blacklist read-only words from the dictionary

### ISSUE THREAD

**Issue by yaomtc** [AnySoftKeyboard/AnySoftKeyboard#1573]
### Steps to reproduce
1. Type "my" in a way that could bring up "NY".
2. Thinking that "NY" may have been something I added on accident, I try to remove it.
3. I can't remove "NY".

### Expected behaviour
Like in SwiftKey, I expect to be able to "remove" this word. I understand that the dictionaries you can get are read-only to AnySoftKeyboard. However I should be able to add "ignore" words to the user dictionary, so that while they can remain in the dictionary file, they will be skipped over for suggestions/corrections. 

### Actual behaviour
Sometimes words/abbreviations come up that I never expect to say, and they're sometimes chosen in place of a word that's more common for the user. Like "my" being intended, but "NY" being typed. Or archaic words that don't fit my style of writing at all, that I'd prefer not to see in the suggestions list again.

**Android OS version:**
9.0 Pie

**Device manufacturer and model:**
Motorola Moto X4

**List of installed add-ons (like languages, or themes):**
English

**Comment by esmoak** [comment]
Would love to see this also. Probably the main thing that keeps me from using ASK full time.

**Comment by forteller** [comment]
Yes! I need this too! I have at least one word that comes up a lot, and that I have to change every single time it does. Every time. I never, ever want to use it, still it's so close to a word I use all the time that I write it on accident all the time. This is very annoying.

I hope it's possible to let us change the default dictionaries soon.

Thank you, and thank you so much for making this important product! :)

**Comment by Kennric** [comment]
Yes please, a thousand times yes. 

I've noticed the corrections appear to not even be weighted by frequency in the language, so often the least useful word is chosen. Swiping "is" results, 100% of the time, in "I's" on my phone, a baffling choice. (See also: words that are also proper nouns seem to be corrected to the proper noun version - "cook" always results in "Cook" for me, which seems to imply that it takes the top choice of a case sensitive alphanumeric sorted list. 

Storing ignorable words in the user dictionary would be nice, storing weighting choice information based on my fixing bad substitutions, would be beautiful.

**Comment by Jirafey** [comment]
can we please ship it? It's the main reason the typing is slow. Why don't you want to complete the only privacy respecting mobile keyboard input method???

type "its" in english you get so many unnecessary words that you will never use, let me clear it to zero, and itll add word by word by usage or ill add them by force or upload the monkeytype dictionaries? or most used words from some database.

plesde don't bury this, at least provide a workaround.

**Comment by yaomtc** [comment]
It's not buried. Right above your comment it shows it's planned for one of the next releases.

There is no workaround. OpenBoard doesn't have the option, nor does any other AOSP based keyboard. FlorisBoard doesn't even have a suggestions bar. Only keyboard I know of on Android that lets you remove/blacklist suggestions is SwiftKey, which is proprietary.

**Comment by Jirafey** [comment]
> It's not buried. Right above your comment it shows it's planned for one of the next releases.
> 
> There is no workaround. OpenBoard doesn't have the option, nor does any other AOSP based keyboard. FlorisBoard doesn't even have a suggestions bar. Only keyboard I know of on Android that lets you remove/blacklist suggestions is SwiftKey, which is proprietary.

but how is this not a priority, I have no idea
